samples/YesSql.Samples.Hi/YesSql.Samples.Hi.csproj

```diff
@@ -16,8 +16,11 @@
     <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <WarningLevel>4</WarningLevel>
```

Comment:
?

Reply:
this is all stuff added by vs4mac preview. I'll have a look and clean it up.